client/src/pages/add-bet/add-stake-form/BetToStake.tsx

```
@@ -44,7 +43,10 @@ export const BetToStake = ({ myBet, handleModifyBet }: BetToStakeProps) => {
             </div>
             <div className="mybet-slip-odds">{Number(bet.odds).toFixed(2)}</div>
             <div className="mybet-slip-more">
-              <a className="mybet-edit" onClick={() => handleModifyBet(index)}>
+              <a
+                className="mybet-edit"
+                onClick={() => handleModifyBet(bet.id as number)}
+              >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)